packages/redux-routine/src/is-action.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { isPlainObject } from 'is-plain-object';
+import type { AnyAction } from 'redux';
```

Comment:
`AnyAction` is deprecated

```suggestion
import type { Action } from 'redux';
```